data/routers/auth-router.js

```diff
@@ -0,0 +1,88 @@
+const router = require('express').Router();
+const bcrypt = require('bcryptjs');
+
+const Auth = require('../models/auth-model');
+
+router.post('/register', async (req, res) => {
+  const { username, password } = req.body;
+  if (!username || !password) {
+    res.status(400).json({
+      error: 'You must provide a username and password to register.'
+    });
+  } else if (username.length > 64 || password.length > 128) {
+    res.status(400).json({
+      error:
+        'Your username may not exceed 64 characters. Your password may not exceed 128 characters.'
+    });
+  } else {
+    try {
+      const checkUsername = await Auth.checkUsername(username);
+      if (checkUsername === 'taken') {
+        res.status(400).json({
+          error: 'That username is already taken. Please try another.'
+        });
+      } else {
+        try {
+          // After declaring const newUser, I wasn't using the variable, so I removed it. It works (including error catching), and I think it looks cleaner. Thoughts?
+          await Auth.registerUser(req.body);
```

Comment:
yes, so long as you don't need to later reference what would be captured in a variable from this await call, there's no need to store it :+1: